app/workers/facilities/access_data_download.rb

```diff
@@ -116,7 +117,7 @@ def parse_wait_time_data(records)
     def update_wait_time_data(client)
       records = client.download
       uniq_specialties = records.map { |facility| facility['ApptTypeName'] }.uniq
-      unless uniq_specialties == WT_KEY_MAP.keys
+      unless uniq_specialties.sort == WT_KEY_MAP.keys.sort
```

Comment:
Is this because these keys come back in a different order from the new endpoint?

Reply:
yep